Files missing from NPM package

When I install `@magenta/music` (1.0.0) with NPM/Yarn, the full contents of the package are as follows:

```
README.md
package.json
dist/.DS_Store
dist/.gitignore
dist/magentamusic.js
es5/core
es5/index.d.ts
es5/index.js
es5/index.js.map
es5/music_rnn
es5/music_vae
es5/protobuf
```

I.e. all the folders under `es5` are empty. 

In a Webpack project this causes the following build error when I try to `import * as mm from '@magenta/music'`:

```
ERROR in ./node_modules/@magenta/music/es5/index.js
Module not found: Error: Can't resolve './core/index' in './node_modules/@magenta/music/es5'
 @ ./node_modules/@magenta/music/es5/index.js 6:9-32
 @ ./src/index.js
 @ multi (webpack)-dev-server/client?http://localhost:8082 ./src/index.js

ERROR in ./node_modules/@magenta/music/es5/index.js
Module not found: Error: Can't resolve './music_rnn/index' in './node_modules/@magenta/music/es5'
 @ ./node_modules/@magenta/music/es5/index.js 7:9-37
 @ ./src/index.js
 @ multi (webpack)-dev-server/client?http://localhost:8082 ./src/index.js

ERROR in ./node_modules/@magenta/music/es5/index.js
Module not found: Error: Can't resolve './music_vae/index' in '.node_modules/@magenta/music/es5'
 @ ./node_modules/@magenta/music/es5/index.js 8:9-37
 @ ./src/index.js
 @ multi (webpack)-dev-server/client?http://localhost:8082 ./src/index.js

ERROR in ./node_modules/@magenta/music/es5/index.js
Module not found: Error: Can't resolve './protobuf/index' in './node_modules/@magenta/music/es5'
 @ ./node_modules/@magenta/music/es5/index.js 9:9-36
 @ ./src/index.js
```

With 1.0.1 there's still one missing file failure from Webpack:

```
ERROR in ./node_modules/@magenta/music/es5/protobuf/index.js
Module not found: Error: Can't resolve './proto' in './node_modules/@magenta/music/es5/protobuf'
 @ ./node_modules/@magenta/music/es5/protobuf/index.js 3:14-32
 @ ./node_modules/@magenta/music/es5/index.js
 @ ./src/index.js
 @ multi (webpack)-dev-server/client?http://localhost:8082 ./src/index.js
ℹ ｢wdm｣: Failed to compile.
```

I tried setting allowJs to true in tsconfig, which seems to work, but I also get:

```
tsconfig.json:7:5 - error TS5053: Option 'allowJs' cannot be specified with option 'declaration'.

7     "declaration": true,
      ~~
```

--- a/music/package.json
+++ b/music/package.json
@@ -32,7 +32,7 @@
   },
   "scripts": {
     "prepublish": "yarn lint && yarn test && yarn build && yarn doc && yarn bundle",
-    "build": "tsc",
+    "build": "tsc && cp src/protobuf/proto.* es5/protobuf",
     "bundle": "browserify --standalone mm src/index.ts -p [tsify] > dist/magentamusic.js",
     "lint": "tslint -c ../tslint.json -p . -t verbose",
     "test": "ts-node node_modules/tape/bin/tape src/**/*_test.ts",

--- a/music/package.json
+++ b/music/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@magenta/music",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "Make music in the browser with machine learning.",
   "main": "es5/index.js",
   "types": "es5/index.d.ts",

